src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -224,6 +225,8 @@ export default class ResultsViewOncoprint extends React.Component<
     @observable selectedHeatmapProfileId = '';
     @observable heatmapGeneInputValue = '';
 
+    @observable selectedGenericAssayProfileId = '';
```

Comment:
i see we do this above, but it's probably not a good idea to use empty string as essentially a null value.   why not just null?